src/Uno.UWP/Extensions/MathEx.cs

```diff
@@ -59,6 +59,14 @@ public static double ToDegreeNormalized(double angleRadian)
 				? ToDegree(angleRadian) % 360
 				: 360 + ToDegree(angleRadian) % 360;
 
+		/// <summary>
+		/// Normalize an angle in degrees in the [0, 360[ range.
+		/// </summary>
+		public static double NormalizeDegree(double angleDegree)
+			=> angleDegree >= 0
+				? angleDegree % 360
+				: 360 + angleDegree % 360;
```

Comment:
What happen with a -800 degrees value?

Reply:
That's teh purpose of the "remainder operator" : https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/arithmetic-operators#remainder-operator-

Reply:
Oooh you're right... `%` is the same priority than `/` and `*`.

Reply:
Oh I see you point ... actually I've put the `360 + ` before to make a bit clearer, but I should probably had some parenthesis to make it obvious.